.travis.yml

```diff
@@ -19,14 +19,14 @@ matrix:
     - php: 5.3
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: PHP_CS_FIXER_VERSION=1.*
+      env: PHP_CS_FIXER_VERSION=2.*
     - php: 5.6
-      env: PHP_CS_FIXER_VERSION=2.*@dev
+      env: PHP_CS_FIXER_VERSION=3.*@dev
```

Comment:
Well, I'll not merge this PR very soon, this is just a preparation.

Let's see what it will be done when php-cs-fixer will be stable.